app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java

```diff
@@ -227,6 +228,7 @@ public void onPDFCreated(boolean success, String path) {
         getSnackbarwithAction(mActivity, R.string.snackbar_pdfCreated)
                 .setAction(R.string.snackbar_viewAction, v -> mFileUtils.openFile(mPath))
                 .show();
+        new DatabaseHelper(getContext()).insertRecord(mPath,"created");
```

Comment:
@vyankatesh24 I think we don't use constant strings directly here. Can you confirm?

Reply:
> @vyankatesh24 I think we don't use constant strings directly here. Can you confirm?

You are right @Swati4star. 

@archi14 

Use `R.string.created` instead of `created`.